examples/react-simple/src/ix/GetAccount.js

```diff
@@ -6,9 +6,21 @@ export const GetAccount = () => {
   const [addr, setAddr] = useState("")
 
   const run = async () => {
-    const response = await sdk.send(await sdk.pipe(await sdk.build([
+
+    /*
+
+      Get Account
+      -----------
+
+      Declaring an interaction which gets an account is done by calling the sdk.getAccount builder.
+      sdk.getAccount consumes an address of an account to get.
```

Comment:
Link or example of Flow account object to demonstrate return value might be helpful here.